sdks/java/core/src/main/java/com/google/cloud/dataflow/sdk/options/ProxyInvocationHandler.java

```diff
@@ -209,6 +261,69 @@ public int hashCode() {
   }
 
   /**
+   * Populate display data. See {@link HasDisplayData#populateDisplayData}.
+   */
+  private void populateDisplayData(DisplayData.Builder builder) {
+    Preconditions.checkState(canPopulateDisplayData(),
+        "Populating display data from deserialized PipelineOptions is not supported.");
+    Set<PipelineOptionsReflector.Property> props =
+        PipelineOptionsReflector.collectVisibleProperties(knownInterfaces);
+
+    Multimap<String, Class<?>> keyToInterfaceMap = buildOptionToInterfaceMap(props);
+
+    for (Map.Entry<String, BoundValue> option : options.entrySet()) {
+      BoundValue boundValue = option.getValue();
+      if (boundValue.isDefault()) {
```

Comment:
Comment here to indicate that for now, we only include non-default values in the display data.